spec/Confirmation_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'Confirmation'
+
+
+describe Confirmation do
+  subject(:confirmation) { described_class.new("+447880521694") }
+
+  before do
+    allow(confirmation).to receive(:send)
+  end
+
+  it "sends a confirmation SMS on checkout" do
+    expect(confirmation).to receive(:send)
+    confirmation.send
```

Comment:
this test will never fail - are you writing test-first? Always watch the test fail before you make it green.